src/firefly/js/visualize/ui/FileUploadViewPanel.jsx

```diff
@@ -619,7 +619,7 @@ function AnalysisTable({summaryModel, detailsModel, report, isMoc, UNKNOWN_FORMA
 
     // Details table needs to render first to create a stub to collect data when Summary table is loaded.
     return (
-        <Stack {...{direction:'row', flexGrow:1, position:'relative', minHeight:'22rem'}}>
+        <Stack {...{direction:'row', flexGrow:1, position:'relative', minHeight:'14rem'}}>
```

Comment:
This is the change that matters the most.  It's what pushing the 'Load' button down when there's not enough vertical space.  I would make this smaller.  It does not need to be big because it will resize based on the browser's size.

Reply:
good point, done. set the `minHeight` to `1rem`. (may be too small, but like you said it resizes and I tested it and looks ok). Test build is updated if you want to have a look. https://fireflydev.ipac.caltech.edu/firefly-1553-upload-popup-bug/firefly